content/millennium/r4/scheduling/appointment.md

```diff
@@ -81,10 +81,15 @@ Notes:
 - The `patient`, `practitioner`, and `location` parameters may be included only once and may not be used in combination.
   For example, `patient=4704007,1316024` is supported but `patient=4704007&patient=1316024` and `patient=4704007&location=633867` are not.
 
-- The `date` parameter may be provided:  
-  - once with a prefix and time component to indicate a specific date/time. (e.g. `&date=ge2019-12-07T22:22:16.270Z`, `&date=lt2019-12-14T22:22:16.270Z`)     
-  - twice with the prefixes `ge` and `lt` to indicate a specific range. The date and prefix pairs must define
-    an upper and lower bound. (e.g. `&date=ge2019-12-07T22:22:16.270Z&date=lt2019-12-14T22:22:16.270Z`)
+- The `date` parameter may be provided when using `patient`, `practitioner`, and `location`.  
+  - Can be provided twice with the prefixes `ge` and `lt` to indicate a specific range. The date and prefix pairs must define
+    an upper and lower bound. (e.g. `&date=ge2019-12-07T22:22:16.270Z`, `&date=lt2019-12-14T22:22:16.270Z`) 
+  - Can also be provided once with a prefix and time component.
+    - If only start date is provided, end date is set to one year after start date.
+      - e.g. `&date=ge2019-12-07T22:22:16.270Z` creates range to `2020-12-07T22:22:16.270Z`
```

Comment:
>- If only start date is provided, end date is set to one year after start date.
      - e.g. `&date=ge2019-12-07T22:22:16.270Z` creates range to `2020-12-07T22:22:16.270Z`

Can we not represent the range in some bracketed form ?
- e.g. `&date=ge2019-12-07T22:22:16.270Z` creates range as `[2019-12-07T22:22:16.270Z, 2020-12-07T22:22:16.270Z]`. Using square brackets to suggest both dates are included.

Reply:
+1, while it should be [2019-12-07T22:22:16.270Z, 2020-12-07T22:22:16.270Z), since the end datetime is excluded